ncbi-amrfinderplus/3.10.36/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM ubuntu:jammy as app
+
+ARG AMRFINDER_VER="3.10.36"
+
+LABEL base.image="ubuntu:jammy"
+LABEL dockerfile.version="1"
+LABEL software="NCBI AMRFinderPlus"
+LABEL software.version="AMRFINDER_VER"
```

Comment:
missing the dollar sign here `$AMRFINDER_VER`

Reply:
That... embarrassing...